external-crates/move/crates/move-compiler/src/shared/program_info.rs

```diff
@@ -206,14 +216,26 @@ impl<const AFTER_TYPING: bool> ProgramInfo<AFTER_TYPING> {
         &self.enum_definition(m, n).type_parameters
     }
 
+    pub fn named_member_kind(&self, m: ModuleIdent, n: Name) -> NamedMemberKind {
```

Comment:
Can remove if we want, but this seems like a generally useful function to have, so vote to keep it.

Reply:
The IDE wants this feature